cli/main.rs

```diff
@@ -382,12 +386,21 @@ async fn bundle_command(
   source_file: String,
   out_file: Option<PathBuf>,
 ) -> Result<(), ErrBox> {
-  let module_name = ModuleSpecifier::resolve_url_or_path(&source_file)?;
+  let mut module_name = ModuleSpecifier::resolve_url_or_path(&source_file)?;
+  let url = module_name.as_url();
+
+  // TODO(bartlomieju): fix this hack in ModuleSpecifier
+  if url.scheme() == "file" {
+    let a = deno_fs::normalize_path(&url.to_file_path().unwrap());
+    let u = Url::from_file_path(a).unwrap();
+    module_name = ModuleSpecifier::from(u)
+  }
```

Comment:
Passing relative path to `ModuleSpecifier::resolve_import()` doesn't remove `./` and `../` parts, we should probably change that